showkase-processor-testing/build.gradle

```diff
@@ -75,9 +81,6 @@ dependencies {
     testImplementation deps.test.assertJ
     testImplementation deps.test.googleTruth
     testImplementation deps.test.junit
-    // Allows this module to access the annotation processor related classes. Otherwise those are
-    // only available in java library modules. Inspiration - 
-    // https://github.com/airbnb/epoxy/blob/master/epoxy-processortest/build.gradle
-    testImplementation files('libs/rt.jar')
```

Comment:
How were we able go get around needing this?

Reply:
the jar was still needed but it was already being pulled in by `implementation fileTree(include: ['*.jar'], dir: 'libs')`

I actually reverted this and deleted the general import though since this is more explicit